management/cmd/management.go

```diff
@@ -162,6 +163,14 @@ var (
 				}
 			}
 
+			geolocationStore, err := geolocation.NewSqliteStore(config.Datadir)
+			if err != nil {
+				log.Warnf("could not  geo location database, we proceed without location endpoints")
+			} else {
+				log.Infof("geo location database has been initialized from %s", config.Datadir)
+			}
+			geolocationManager := geolocation.NewManager(geolocationStore)
```

Comment:
Maybe the `geolocationManager` should be a part of `geolocation.NewGeolocation` then we don't need to have two objects with similar scope.